README.md

```diff
@@ -1,313 +1,358 @@
-
-commands:
-  {collect,forward,run}
-    collect             Collects the logs from different processes and sends
-                        to nsq
-    forward             Collects all the messages from nsq and pushes to
-                        storage engine
-```
+    ```bash
+    $ logagg --help
+    ```
+    ##### or
+    ```
+    $ sudo docker run deepcompute/logagg logagg --help
+    ```
+    - you should see something like:
+    ```
+    usage: logagg [-h] [--name NAME] [--log-level LOG_LEVEL]
+                  [--log-format {json,pretty}] [--log-file LOG_FILE] [--quiet]
+                  {collect,forward,run} ...
+
+    Logagg command line tool
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      --name NAME           Name to identify this instance
+      --log-level LOG_LEVEL
+                            Logging level as picked from the logging module
+      --log-format {json,pretty}
+                            Force the format of the logs. By default, if the
+                            command is from a terminal, print colorful logs.
+                            Otherwise print json.
+      --log-file LOG_FILE   Writes logs to log file if specified, default: None
+      --quiet               if true, does not print logs to stderr, default: False
+
+    commands:
+      {collect,forward,run}
+        collect             Collects the logs from different processes and sends
+                            to nsq
+        forward             Collects all the messages from nsq and pushes to
+                            storage engine
+    ```
 
 - For `logagg collect`
-```bash
-logagg collect --help
-```
-- You should see something like
-```bash
-usage: logagg collect [-h] [--nsqchannel NSQCHANNEL]
-                      [--nsqd-http-address NSQD_HTTP_ADDRESS]
-                      [--depth-limit-at-nsq DEPTH_LIMIT_AT_NSQ]
-                      [--exception-logs-file EXCEPTION_LOGS_FILE]
-                      file [file ...] nsqtopic
-
-positional arguments:
-  file                  Provide absolute path of log file including the module name
-                        and function name, eg: /var/log/nginx/access.log:logag
-                        g.collect.handlers.nginx_access
-  nsqtopic              Topic name to publish messages. Ex: logs_and_metrics
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --nsqchannel NSQCHANNEL
-                        Channel of nsqd
-  --nsqd-http-address NSQD_HTTP_ADDRESS
-                        nsqd HTTP address where we send the messages
-  --depth-limit-at-nsq DEPTH_LIMIT_AT_NSQ
-                        To limit the depth at nsq channel
-  --exception-logs-file EXCEPTION_LOGS_FILE
-                        If collector fails to publish messages to nsq, will
-                        write the logs to a file
-```
+    ```bash
+    $ logagg collect --help
+    ```
+    ##### or
```

Comment:
@supriyopaul  Hashes are a mechanism to specify header levels. Is this required here?